Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2734 @@
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint='/atpapi/v2/auditevents',
+            params={},
+            json_data=payload
+        ).json()
+
+    def get_event_list(self, payload: dict) -> dict[str, Any]:
+        """
+        Client method for get Events List
+        Args:
+            payload (dict): request json body
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint='/atpapi/v2/events',
+            params={},
+            json_data=payload
+        ).json()
+
+    def get_system_activity(self, payload: dict) -> dict[str, Any]:
+        """
+        Client method for get System Activity
+        Args:
+            payload (dict): request json body
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint='/atpapi/v2/systemactivities',
+            params={},
+            json_data=payload
+        ).json()
+
+    def get_event_for_incident(self, payload: dict) -> dict[str, Any]:
+        """
+        Client method for get event for Incident
+        Args:
+            payload (dict): request json body
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint='/atpapi/v2/incidentevents',
+            params={},
+            json_data=payload
+        ).json()
+
+    def get_incident(self, payload: dict) -> dict[str, Any]:
+        """
+        Client method for get Incident
+        Args:
+            payload (dict): request json body
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint='/atpapi/v2/incidents',
+            params={},
+            json_data=payload
+        ).json()
+
+    def get_incident_comment(self, payload: dict, uuid: str) -> dict[str, Any]:
+        """
+        Client method for get Incident
+        Args:
+            payload (dict): request json body
+            uuid (str): Incident Unique ID
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint=f'/atpapi/v2/incidents/{uuid}/comments',
+            params={},
+            json_data=payload
+        ).json()
+
+    def add_incident_comment(self, uuid: str, value: str) -> requests.Response:
+        """
+        Client method for add Incident comment
+
+        Args:
+            uuid : Incident Unique ID
+            value : Incident Comment
+        Returns:
+            return response json
+        """
+        request_data: list[dict[str, Any]] = [
+            {'op': 'add', 'path': f'/{uuid}/comments', 'value': value[:512]}
+        ]
+        # json.dumps([payload])
```

Comment:
?

Reply:
Kindly refer to my comments .. https://github.com/demisto/content/pull/23280#discussion_r1108027662

Reply:
Though, I have modified it now and add ValueError exception if comment length is more than 512 characters.

Reply:
line 401 seems to be leftover, commented-out code

Reply:
line 401 removed.